lib/matplotlib/backends/backend_wx.py

```diff
@@ -1285,12 +1285,6 @@ def __init__(self, canvas, num, frame):
         self.tb = frame.GetToolBar()
         self.toolbar = self.tb  # consistent with other backends
 
-        def notify_axes_change(fig):
-            'this will be called whenever the current axes is changed'
-            if self.tb is not None:
-                self.tb.update()
```

Comment:
Won't this cause problems here because the tool bar is `self.tb` and not `self.toolbar` like all the other backends?

Reply:
It is already aliased to toolbar too (in fact I'm getting rid of the tb alias in a separate PR).